[gdb] unable to install

checking whether to enable maintainer-specific portions of Makefiles... no
configure: creating ./config.status
config.status: creating Makefile
**gdb failed to install: `make V=0 -j4` exited with 127**

On aarch64:

`g++ -x c++ -std=gnu++11 -g -O2   -I. -I. -I./common -I./config -DLOCALEDIR="\"/usr/local/share/locale\"" -DHAVE_CONFIG_H -I./../include/opcode -I./../opcodes/.. -I./../readline/.. -I./../zlib -I../bfd -I./../bfd -I./../include -I../libdecnumber -I./../libdecnumber  -I./gnulib/import -Ibuild-gnulib/import   -DTUI=1  -I/usr/local/include/python2.7 -I/usr/local/include/python2.7 -Wall -Wpointer-arith -Wno-unused -Wunused-value -Wunused-function -Wno-switch -Wno-char-subscripts -Wempty-body -Wunused-but-set-parameter -Wunused-but-set-variable -Wno-sign-compare -Wno-narrowing -Wno-error=maybe-uninitialized  -c -o aarch64-linux-hw-point.o -MT aarch64-linux-hw-point.o -MMD -MP -MF ./.deps/aarch64-linux-hw-point.Tpo nat/aarch64-linux-hw-point.c
In file included from nat/aarch64-linux-hw-point.c:26:0:
/usr/local/include/sys/ptrace.h:75:4: error: expected identifier before numeric constant
    PTRACE_GETREGS = 12,
    ^
/usr/local/include/sys/ptrace.h:75:4: error: expected ‘}’ before numeric constant
/usr/local/include/sys/ptrace.h:75:4: error: expected unqualified-id before numeric constant
In file included from /usr/local/include/features.h:368:0,
                 from /usr/local/include/stdio.h:27,
                 from build-gnulib/import/stdio.h:43,
                 from ./common/common-defs.h:52,
                 from nat/aarch64-linux-hw-point.c:19:
/usr/local/include/sys/ptrace.h:225:1: error: expected declaration before ‘}’ token
 __END_DECLS
 ^
make[2]: *** [Makefile:1640: aarch64-linux-hw-point.o] Error 1
make[2]: *** Waiting for unfinished jobs....
In file included from ./common/common-defs.h:79:0,
                 from defs.h:28,
                 from aarch64-linux-nat.c:21:
aarch64-linux-nat.c: In function ‘void fetch_fpregs_from_thread(regcache*)’:
./common/gdb_assert.h:25:33: error: static assertion failed: 
 #define gdb_static_assert(expr) static_assert (expr, "")
                                 ^
aarch64-linux-nat.c:245:3: note: in expansion of macro ‘gdb_static_assert’
   gdb_static_assert (sizeof regs >= VFP_REGS_SIZE);
   ^
aarch64-linux-nat.c:273:15: error: ‘elf_fpregset_t’ has no member named ‘vregs’
         &regs.vregs[regno - AARCH64_V0_REGNUM]);
               ^
aarch64-linux-nat.c:275:65: error: attempt to take address of bit-field structure member ‘user_fpregs::fpsr’
       regcache_raw_supply (regcache, AARCH64_FPSR_REGNUM, &regs.fpsr);
                                                                 ^
aarch64-linux-nat.c:276:65: error: attempt to take address of bit-field structure member ‘user_fpregs::fpcr’
       regcache_raw_supply (regcache, AARCH64_FPCR_REGNUM, &regs.fpcr);
                                                                 ^
In file included from ./common/common-defs.h:79:0,
                 from defs.h:28,
                 from aarch64-linux-nat.c:21:
aarch64-linux-nat.c: In function ‘void store_fpregs_to_thread(const regcache*)’:
./common/gdb_assert.h:25:33: error: static assertion failed: 
 #define gdb_static_assert(expr) static_assert (expr, "")
                                 ^
aarch64-linux-nat.c:293:3: note: in expansion of macro ‘gdb_static_assert’
   gdb_static_assert (sizeof regs >= VFP_REGS_SIZE);
   ^
aarch64-linux-nat.c:321:20: error: ‘elf_fpregset_t’ has no member named ‘vregs’
     (char *) &regs.vregs[regno - AARCH64_V0_REGNUM]);
                    ^
aarch64-linux-nat.c:325:25: error: attempt to take address of bit-field structure member ‘user_fpregs::fpsr’
          (char *) &regs.fpsr);
                         ^
aarch64-linux-nat.c:328:25: error: attempt to take address of bit-field structure member ‘user_fpregs::fpcr’
          (char *) &regs.fpcr);
                         ^
make[2]: *** [Makefile:1621: aarch64-linux-nat.o] Error 1
make[2]: Leaving directory '/usr/local/tmp/crew/gdb-8.1.tar.xz.dir/gdb-8.1/gdb'
make[1]: *** [Makefile:9127: all-gdb] Error 2
make[1]: Leaving directory '/usr/local/tmp/crew/gdb-8.1.tar.xz.dir/gdb-8.1'
make: *** [Makefile:850: all] Error 2
gdb failed to install: `make V=0 -j4` exited with 2
`

--- a/packages/alive.rb
+++ b/packages/alive.rb
@@ -8,8 +8,16 @@ class Alive < Package
   source_sha256 '120dd9ef361833623be353ad8cfac585abae51a16fedd3a84f1d99a842793fef'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/alive-2.0.2-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/alive-2.0.2-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/alive-2.0.2-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/alive-2.0.2-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: '02f36ed0016450c778347646a793d743eeb96c5f9cfac3652e53a878eba6a505',
+     armv7l: '02f36ed0016450c778347646a793d743eeb96c5f9cfac3652e53a878eba6a505',
+       i686: '7b952c08c377e4888ef62bed54c06d051258dfb5b4734033e4a80f8616bae256',
+     x86_64: '82bbdaf0fb8e1bcd3763900536caa84dd043ed9296d0e038da7a9f96801716fd',
   })
 
   depends_on 'guile'

--- a/packages/gdb.rb
+++ b/packages/gdb.rb
@@ -8,8 +8,16 @@ class Gdb < Package
   source_sha256 'af61a0263858e69c5dce51eab26662ff3d2ad9aa68da9583e8143b5426be4b34'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/gdb-8.1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/gdb-8.1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/gdb-8.1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gdb-8.1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: '0487578f13f93e89ad87ec45a1468e057055465a83de7ee8a1cd27a4b467732b',
+     armv7l: '0487578f13f93e89ad87ec45a1468e057055465a83de7ee8a1cd27a4b467732b',
+       i686: 'e4078ff2566028ca81aa00ee3b47bfc4723fa8d75422fc80b463e8273844aef0',
+     x86_64: '8e7f25bb702e9bdf3b1d657aee7cf79e6de0cfdcf6bb512159818c02e39e2593',
   })
 
   depends_on "ncurses"

--- a/packages/gox.rb
+++ b/packages/gox.rb
@@ -7,6 +7,19 @@ class Gox < Package
   source_url 'https://github.com/mitchellh/gox/archive/v0.4.0.tar.gz'
   source_sha256 '2df7439e9901877685ff4e6377de863c3c2ec4cde43d0ca631ff65d1b64774ad'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/gox-0.4.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/gox-0.4.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/gox-0.4.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gox-0.4.0-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'bd18d1ee7da98692c0aaa53e9016d16a7a2cb5ecf07236c340a2613b6837f087',
+     armv7l: 'bd18d1ee7da98692c0aaa53e9016d16a7a2cb5ecf07236c340a2613b6837f087',
+       i686: '70e31b0602ad26369b711a0f740391d43cd297b4a5d16c4d1a04bdb843f99fb0',
+     x86_64: 'd6d6c6c6d5aa168bea5c25a202c902d8b965040218a52291701c3e172d29c991',
+  })
+
   depends_on 'go'
 
   def self.install

--- a/packages/heroku.rb
+++ b/packages/heroku.rb
@@ -7,6 +7,19 @@ class Heroku < Package
   source_url 'https://github.com/heroku/cli/archive/v6.16.13.tar.gz'
   source_sha256 '70be5c06f541f30d768bd3bb486db73a8e5873f136fd3e23b69717f9cca6246a'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/heroku-6.16.13-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/heroku-6.16.13-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/heroku-6.16.13-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/heroku-6.16.13-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'a38a8ab189e8ae52b268a114fbdb25b759b436085327312436c4e023b0e55b47',
+     armv7l: 'a38a8ab189e8ae52b268a114fbdb25b759b436085327312436c4e023b0e55b47',
+       i686: '2c8c80ea39a22abfd81cfe4ad751ee9578dd2818a01050dfe5b09acf9b18028a',
+     x86_64: '23b38b78a9ff83a0a2b77a908f3616e1ccedbaa44c0a474f21dd40f2d1eabeaa',
+  })
+
   depends_on 'yarn'
 
   def self.build

--- a/packages/libsdl.rb
+++ b/packages/libsdl.rb
@@ -3,26 +3,33 @@ require 'package'
 class Libsdl < Package
   description 'Simple DirectMedia Layer is a cross-platform development library designed to provide low level access to audio, keyboard, mouse, joystick, and graphics hardware via OpenGL and Direct3D.'
   homepage 'http://www.libsdl.org'
-  version '1.2.15-1'
+  version '1.2.15-2'
   source_url 'http://www.libsdl.org/release/SDL-1.2.15.tar.gz'
   source_sha256 'd6d316a793e5e348155f0dd93b979798933fb98aa1edebcc108829d6474aad00'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libsdl-1.2.15-1-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libsdl-1.2.15-1-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libsdl-1.2.15-1-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libsdl-1.2.15-1-chromeos-x86_64.tar.xz',
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libsdl-1.2.15-2-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libsdl-1.2.15-2-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libsdl-1.2.15-2-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libsdl-1.2.15-2-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: '7243fc79a7734a63ce5a6496dc1a6a7b2a9f22c2f8e96f7ec7ec437144835cd6',
-     armv7l: '7243fc79a7734a63ce5a6496dc1a6a7b2a9f22c2f8e96f7ec7ec437144835cd6',
-       i686: 'da4d8372f7954e9da9c02734af6dbb5238917fcc0b1e5b08803f807a0683ab27',
-     x86_64: '0237c7da8ab2f5f636881bfbe7a858799b2bb05d8e26ff411305680ac69aba29',
+    aarch64: 'dd572e285fc4485ec46c36f7612f06f78ee2565e4fa5814c2828c520ea7f44ac',
+     armv7l: 'dd572e285fc4485ec46c36f7612f06f78ee2565e4fa5814c2828c520ea7f44ac',
+       i686: '485362d999a6a6ab466c8a544df175b9e93db2b5149ee0f37be4bf33d6adc66b',
+     x86_64: 'f9c9cee413c6acd449fefadb44f8148595124d994b658fedca67642921369b43',
   })
 
+  depends_on 'dbus'
+  depends_on 'eudev'
+
   def self.build
-    system "./configure --disable-video-x11 --disable-video-wayland"
-    system "make"
+    system './configure',
+           "--prefix=#{CREW_PREFIX}",
+           "--libdir=#{CREW_LIB_PREFIX}",
+           '--disable-video-wayland',
+           '--disable-video-x11'
+    system 'make'
   end
 
   def self.install

--- a/packages/libsdl2.rb
+++ b/packages/libsdl2.rb
@@ -3,26 +3,32 @@ require 'package'
 class Libsdl2 < Package
   description 'Simple DirectMedia Layer is a cross-platform development library designed to provide low level access to audio, keyboard, mouse, joystick, and graphics hardware via OpenGL and Direct3D.'
   homepage 'http://www.libsdl.org'
-  version '2.0.5-1'
-  source_url 'https://www.libsdl.org/release/SDL2-2.0.5.tar.gz'
-  source_sha256 '442038cf55965969f2ff06d976031813de643af9c9edc9e331bd761c242e8785'
+  version '2.0.8'
+  source_url 'https://www.libsdl.org/release/SDL2-2.0.8.tar.gz'
+  source_sha256 'edc77c57308661d576e843344d8638e025a7818bff73f8fbfab09c3c5fd092ec'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libsdl2-2.0.5-1-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libsdl2-2.0.5-1-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libsdl2-2.0.5-1-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libsdl2-2.0.5-1-chromeos-x86_64.tar.xz',
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libsdl2-2.0.8-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libsdl2-2.0.8-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libsdl2-2.0.8-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libsdl2-2.0.8-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: '79597d7b228ebb8929ceef776032226f9fa9d70672f40b421447b964469da2f0',
-     armv7l: '79597d7b228ebb8929ceef776032226f9fa9d70672f40b421447b964469da2f0',
-       i686: 'b242dc03f077e0758336956bcc28e935773f75cca0720d047d1d8f3be94cb8c3',
-     x86_64: '84d257d6f385e4535f8643d6dc0c5dc455fa197705ff17e058024822f0fe7f86',
+    aarch64: '8d72ba834b2b3134f31b48f50febba28063da56f1e4dea97baffe1eca55b337d',
+     armv7l: '8d72ba834b2b3134f31b48f50febba28063da56f1e4dea97baffe1eca55b337d',
+       i686: '1c54d9a41d86ceeb78fa6b2f688ae886d5aa00c694e30cce173bd5ec8444195f',
+     x86_64: 'c771bddb8be9b2637978593a013f880f6faeea651d93c66fdb92f38c30100580',
   })
 
+  depends_on 'dbus'
+  depends_on 'eudev'
+
   def self.build
-    system "./configure --disable-video-x11 --disable-video-wayland"
-    system "make"
+    system './configure',
+           "--prefix=#{CREW_PREFIX}",
+           "--libdir=#{CREW_LIB_PREFIX}",
+           '--disable-video-wayland',
+           '--disable-video-x11'
   end
 
   def self.install

--- a/packages/yarn.rb
+++ b/packages/yarn.rb
@@ -3,21 +3,21 @@ require 'package'
 class Yarn < Package
   description 'Yarn is a new package manager for JavaScript and an alternative to npm.'
   homepage 'https://yarnpkg.com/en/'
-  version '1.3.2-20171127.1833'
-  source_url 'https://nightly.yarnpkg.com/yarn-v1.3.2-20171127.1833.tar.gz'
-  source_sha256 '36293f1327693a8742a9030aa8b8213eb7c33b298b7eb28d2d9b2efe93279060'
+  version '1.6.0'
+  source_url 'https://github.com/yarnpkg/yarn/releases/download/v1.6.0/yarn-v1.6.0.tar.gz'
+  source_sha256 'a57b2fdb2bfeeb083d45a883bc29af94d5e83a21c25f3fc001c295938e988509'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/yarn-1.3.2-20171127.1833-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/yarn-1.3.2-20171127.1833-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/yarn-1.3.2-20171127.1833-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/yarn-1.3.2-20171127.1833-chromeos-x86_64.tar.xz',
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/yarn-1.6.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/yarn-1.6.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/yarn-1.6.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/yarn-1.6.0-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: '14ec12edc8ef9e74a0fc7bf914e3e09f2eb90a70ef5a32bf858fe24dd42dcbcd',
-     armv7l: '14ec12edc8ef9e74a0fc7bf914e3e09f2eb90a70ef5a32bf858fe24dd42dcbcd',
-       i686: 'fe3ac396402804f42e9d74bf36950a6fed8df5b3a8b6e50c666b8c0ff7a69b1f',
-     x86_64: '2f3c453523d498376bf5182fb01eaebb1f269dd32cb8dae40eb7515386dec3e6',
+    aarch64: '3c85013806518b087950a93ff2b5bdbc986a79dd56f93063a24eff1d785657f7',
+     armv7l: '3c85013806518b087950a93ff2b5bdbc986a79dd56f93063a24eff1d785657f7',
+       i686: '53303cb77b189f5a166d2b3efaac3e840c89f61ea49bf9ab6fc3b7bf724c7bc1',
+     x86_64: 'db01d493bcc9277d259681806871eeb99161ce2857fb5843ab880d966cafd5fb',
   })
 
   node_version=`node -v 2> /dev/null`

